addons/project/views/project_sharing_views.xml

```diff
@@ -207,7 +207,7 @@
                             <field name="allow_subtasks" invisible="1" />
                             <field name="partner_id" options="{'no_open': True, 'no_create': True, 'no_edit': True}"/>
                             <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}"/>
-                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True, 'no_edit_color': True}"/>
+                            <field name="tag_ids" context="{'project_id': project_id}" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True, 'no_edit_color': True}"/>
```

Comment:
I think we have to fix it in saas-15.3, since the task 2735833 adds it in the task view used in backend. 🙂

Reply:
@xavierbol , Right, changed target to 15.3 Thanks

Reply:
Great thanks. 🙂